### ISSUE TITLE
Create ScrollableContent element

### ISSUE THREAD

**Issue by cezaraugusto** [brave/browser-laptop#8505]
Issues #8475 and #5076 share almost the same fix: for content longer than viewport users should be able to scroll.

Since it's likely to have more issues regarding that, we should have a component carrying a styled scrollbar to make it consistent across our app.
